roles/readlater/templates/var_www_wallabag_app_config_parameters.yml.j2

```diff
@@ -0,0 +1,29 @@
+parameters:
+    database_driver: pdo_pgsql
+    database_host: localhost
+    database_port: ~
+    database_name: {{ wallabag_db_database }}
```

Comment:
Now I get what you are after! I think there is no default config in the wallabag source code, as it is generated during installation time. But I'll find out the default values